packages/edit-site/src/components/sidebar/global-styles/preview.js

```diff
@@ -25,7 +25,7 @@ export const StylePreview = () => {
 			style={ { background: backgroundColor } }
 		>
 			<HStack spacing={ 5 }>
-				<div>
+				<div style={ { lineHeight: '1' } }>
```

Comment:
Can we apply it in CSS maybe to this class `edit-site-global-styles-preview` or by adding another class here `edit-site-global-styles-preview__text`?

Reply:
We can, yes, I'm a bit confused at myself for adding it here in the first place. Good call.